R/sparklyr.sedona/man/sedona_read_geojson_to_typed_rdd.Rd

```diff
@@ -0,0 +1,64 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
For the documentation, please do not include generated docs in the PR. Please include the source code of docs in the PR. Generated R docs will be manually integrated with generated html files (based on incubator-sedona/docs/) and uploaded to https://github.com/apache/incubator-sedona-website/tree/asf-site

I believe the R docs itself could be a separate PR since it requires some structure change or doc change.